Dockerfile

```
@@ -19,14 +20,24 @@ RUN \
     bzip2 \
     fontconfig \
     libpng-dev \
-    sudo \
-    openjdk-11-jdk && \
-  update-java-alternatives -s java-1.11.0-openjdk-amd64 && \
-  # install node.js
-  wget https://nodejs.org/dist/v14.16.0/node-v14.16.0-linux-x64.tar.gz -O /tmp/node.tar.gz && \
+    sudo && \
+  ARCH="$(dpkg --print-architecture)"; \
+  case "${ARCH}" in \
+     aarch64|arm64) \
+       ARCH='arm64'; \
+       ;; \
+     amd64|x86_64) \
+       ARCH='x64'; \
+       ;; \
+     *) \
+       echo "Unsupported arch: ${ARCH}"; \
+       exit 1; \
+       ;; \
+  esac; \
+  wget https://nodejs.org/dist/v14.17.6/node-v14.17.6-linux-$ARCH.tar.gz -O /tmp/node.tar.gz && \
   tar -C /usr/local --strip-components 1 -xzf /tmp/node.tar.gz && \
   # upgrade npm
-  npm install -g npm && \
+  npm install -g npm@latest && \
```

@mshima Why use latest version of npm?
Better to use lts, no?

---

```suggestion
  npm install -g npm@^7 && \
```
`npm install -g npm` doesn’t update npm currently, don’t remember npm@6 behavior.
Just update from npm 6 to 7 is enough.